temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -214,6 +222,33 @@ public Builder setDefaultDeadlockDetectionTimeout(long defaultDeadlockDetectionT
       return this;
     }
 
+    /**
+     * @param interval the maximum amount of time between sending each pending heartbeat to the
+     *     server. Regardless of heartbeat timeout, no pending heartbeat will wait longer than this
+     *     amount of time to send. Default is 60s, which is chosen if set to null or 0.
+     * @return {@code this}
+     */
+    public Builder setMaxHeartbeatThrottleInterval(Duration interval) {
+      Preconditions.checkArgument(
+          !interval.isNegative(), "Negative maxHeartbeatThrottleInterval value: " + interval);
```

Comment:
Docs say "if set to null". If you're implying that null can be passed as a parameter, this will throw an NPE.